crates/ruff_linter/src/rules/pydocstyle/rules/sections.rs

```diff
@@ -261,9 +258,6 @@ impl AlwaysFixableViolation for SectionUnderlineNotOverIndented {
 ///         raise FasterThanLightError from exc
 /// ```
 ///
-/// ## Options
-/// - `lint.pydocstyle.convention`
-///
 /// ## References
```

Comment:
We should keep this link, PEP257 disables this rule

Reply:
I have returned the link to the setting and added more details on the effect the setting has on this rule.